src/dappsExplorer/DappRankings.tsx

```diff
@@ -1,13 +1,14 @@
 import React, { RefObject, useEffect } from 'react'
 import { useTranslation } from 'react-i18next'
 import { StyleSheet, Text, View } from 'react-native'
+import Toast from 'react-native-simple-toast'
```

Comment:
Shouldn't we use our custom toast instead?

Reply:
i checked with Reed about this shortcut and he was okay with it. the custom toast we have isn't styled in the way the designs were, and expects an action which this design doesn't have